en/docs/InstallAndSetup/PerfromanceTuningAndTestResults/tuning-performance.md

```diff
@@ -270,42 +261,48 @@ Make the number of threads equal to the number of processor cores.</td>
+<p>The global timeout is defined in the <code>                    &lt;API-M_HOME&gt;/repository/conf/synapse.properties                   </code> file. The recommended value is 120000 ms.</p>
+<div class="admonition danger">
+     <p class="admonition-title">Not supported</p>
+     <p>
+     Cannot configure using deployment.toml</p>
+     </div> 
+ </td>
+</td>
 </tr>
 <tr class="even">
-<td><code>                   Endpoint-level timeout                  </code></td>
+<td><code style="white-space: nowrap;"><br />Endpoint-level <br />timeout</code></td>
 <td><div class="content-wrapper">
 <p>You can define timeouts per endpoint for different backend services, along with the action to be taken in case of a timeout.</p>
-<p>The example below sets the endpoint to 50 seconds (50000 ms) and executes the fault handler in case of a timeout.</p>
-<div class="code panel pdl" style="border-width: 1px;">
-<div class="codeContent panelContent pdl">
-<pre class="xml" data-syntaxhighlighter-params="brush: xml; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: xml; gutter: false; theme: Confluence"><code>&lt;timeout&gt;
-   &lt;duration&gt;50000&lt;/duration&gt;
-   &lt;responseAction&gt;fault&lt;/responseAction&gt;
-&lt;/timeout&gt;</code></pre>
-</div>
-</div>
-<p>Alternatively, you can set this through the Publisher UI as well, by following the steps below:</p>
+<p>You can set this through the Publisher UI by following the steps below:</p>
 <ol>
-<li>Log in to the API Publisher ( <code>                      https://&lt;HostName&gt;:9443/publisher                     </code> ). Select your API and click <strong>Edit API</strong> .</li>
-<li>Click the <strong>Implement</strong> tab and click the cogwheel icon next to the endpoint you want to re-configure.</li>
+<li>Log in to the API Publisher ( <code>                      https://&lt;HostName&gt;:9443/publisher                     </code> ). Select your API and click <strong>Endpoints</strong> .</li>
+<li>Click the cogwheel icon next to the endpoint you want to re-configure.</li>
 <li><p>In the <strong>Advanced Settings</strong> dialog box that appears, increase the duration by modifying the default property set as 3000 ms.</p>
-<p>!!! note</p>
-<p>Note that when the endpoint is suspended, the default action is defined here as invoking the fault sequence.</p>
-</p>
-<p><img src="attachments/50518314/80711443.png" alt="timeout.png" width="734" /></p></li>
-<li><p>Click <strong>Save</strong> and re-publish the API.</p></li>
+
+<div class="admonition note">
+     <p class="admonition-title">Note</p>
+     <p>Note that when the endpoint is suspended, the default action is defined here as invoking the fault sequence.</p>
+</div> 
+<p><img src="../../../../assets/attachments/103333648/103333649.png" alt="timeout.png" width="734" /></p></li>
```

Comment:
Image path needs to be updated and the image needs to be moved out of the assets/attachments/ folder as mentioned before.

Reply:
fixed